### PR TITLE
add the pure-white color for the prompt header text

### PR DESCRIPTION
**Pull request by pochretien**

### CODE REVIEW

File: publish-pages.sh [hunk 1]
```diff
@@ -4,8 +4,8 @@ set -x
 
 ( cd _gh_pages
   git init
-  git config user.name "pastjean"
-  git config user.email "pastjean@coveo.com"
+  git config user.name "pochretien"
+  git config user.email "pochretien@coveo.com"
```
Comment (sroy3) [inline publish-pages.sh]: Why do you have to change the user for it to work?

Comment (pochretien) [inline publish-pages.sh]: I simply added someone from the team working on it

Comment (GermainBergeron) [inline publish-pages.sh]: We should probably ask devbooster for a user/user email for this

Comment (MichelML) [inline publish-pages.sh]: @sroy3 personally it was not working when I tried it last time with pastjean

**Pull request merged**